t/mutual-ssl.t

```diff
@@ -110,3 +110,110 @@ Fk5co20gRUsNvK7UWswr9VF7O+5AbHIcdKxIXJj4tECEdnkeJMNSPuD4/KMWRT2t
 wmruxZNnoWGoUeF/w7VBAkEAz02rLKfeCShkbJd5f0qx0cubG/2lznrMvSBoI2ix
 itRV3G/wOHrsK9k06jx2L/+/YTUWcyzTm8B7Y34CTHGaHQ==
 -----END RSA PRIVATE KEY-----
+
+=== TEST 2: Mutual SSL with password file
+--- main_config
+env APICAST_PROXY_HTTPS_PASSWORD_FILE=$Test::Nginx::Util::ServRoot/html/passwords.file;
```

Comment:
We do not use `main_config` because this is a blackbox test. 
You can use `env` block like https://github.com/3scale/apicast/blob/620c1b269fafa29068e35b93c0c3e83de4ccf920/t/http-proxy.t#L15-L18